### PR TITLE
housekeeping: upgraded eventbuilder to vs2017 project format

### PR DESCRIPTION
**Pull request by ghuntley**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

housekeeping/chore/bugfix

**What is the current behavior? (You can also link to an open issue here)**

After we upgraded ReactiveUI to netstandard I've been unable to F5 debug EventBuilder.  i.e. "Build -> Run"

**What is the new behavior (if this is a feature change)?**

Tried for for 30 mins to resolve the issue but ended up fighting with the Microsoft toolchain. Rather than waste any more time I burned the existing csproj and created a new VS2017 version. I can now F5 debug EventBuilder.

**Does this PR introduce a breaking change?**

No

**Please check if the PR fulfills these requirements**
- [ ] The commit follows our guidelines: https://github.com/reactiveui/reactiveui#contribute
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: src/EventBuilder/EventBuilder.csproj [hunk 1]
```diff
@@ -1,130 +1,19 @@
...
-      <Version>1.15.3.7</Version>
-    </PackageReference>
-    <PackageReference Include="Polly">
-      <Version>3.0.0</Version>
-    </PackageReference>
-    <PackageReference Include="Serilog">
-      <Version>1.5.14</Version>
-    </PackageReference>
+
+<ItemGroup>
```
Comment (olevett) [inline src/EventBuilder/EventBuilder.csproj]: Should be able to also 🔥 the nuget package file.

Comment (olevett) [inline src/EventBuilder/EventBuilder.csproj]: Also, we might be able to pull the nuget restore into an MSBuild command and slightly simplify the cake file?

File: build.cake [hunk 2]
```diff
@@ -123,8 +123,8 @@ Task("GenerateEvents")
     .IsDependentOn("BuildEventBuilder")
     .Does (() =>
 {
-    var eventBuilder = "./src/EventBuilder/bin/Release/EventBuilder.exe";
-    var workingDirectory = "./src/EventBuilder/bin/Release";
+    var eventBuilder = "./src/EventBuilder/bin/Release/net45/EventBuilder.exe";
+    var workingDirectory = "./src/EventBuilder/bin/Release/Net45";
```
Comment (olevett) [inline build.cake]: Given the build failing on AppVeyor, would it be worth us dumping some of the logging output to the console around here to make it easier to debug?

Comment (ghuntley) [inline build.cake]: Yup; good idea.

**Comment by ghuntley** [comment]
Fixed in #1410

**Pull request closed without merging**